src/Altinn.Platform/Altinn.Platform.Storage/Storage/Repository/DataRepository.cs

```diff
@@ -33,6 +34,11 @@ public class DataRepository : IDataRepository
         private readonly CloudBlobClient _blobClient;
         private readonly CloudBlobContainer _container;
 
+        /// <summary>
+        /// Gets or sets the data context for the application owner
+        /// </summary>
+        public OrgDataContext OrgDataContext { get; set; }
```

Comment:
consider creating the context for each method that uses it

Reply:
Short term solution ended up being setting Data and InstanceRepository to Transient.